edgelet/edgelet-docker/src/settings.rs

```diff
@@ -33,6 +34,7 @@ pub struct MobyRuntime {
     #[serde(with = "url_serde")]
     uri: Url,
     network: MobyNetwork,
+    content_trust_root_ca_registry: Option<std::collections::HashMap<String, PathBuf>>,
```

Comment:
Add an import for `std::collections::HashMap` (to the existing `std::collections` import on line 3) instead of using the fully-qualified name.